run-checks

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
Curious, what bash specific feature are we using in here?

Reply:
`            echo "- $snap_yaml"`

Apparently this is ambiguous, as `-` is used to denote flags. Shellcheck raises this as a warning with /bin/sh. It's still ambiguous wrt. the flags, but at least shellcheck is now silent.

IMO a proper, unambiguous version would be `printf "- $snap_yaml\n"`, but that looks just awkward.

Reply:
I slightly prefer printf over making this bash specific. Yes, its slightly more ugly and a bit unusual but given how non-portable echo is, maybe the world needs to adjust to seeing more printf in shell scripts :)

Reply:
Ok. FWIW here's how `echo` behaves (note that `%` is just zsh way of saying there's no newline):

```
maciek@corsair:~ /bin/echo "-n" "foo"
foo%
maciek@corsair:~ /bin/echo "-N" "foo"
-N foo
```

Reply:
Pushed a workaround. We can avoid using printfs and still keep the offending pieces readable.

Reply:
Nice! Now we can revert the first line back to /bin/sh, right?